osu.Game/Beatmaps/WorkingBeatmap.cs

```diff
@@ -208,6 +208,9 @@ private Task<IBeatmap> loadBeatmapAsync()
             {
                 return beatmapLoadTask ??= Task.Factory.StartNew(() =>
                 {
+                    // Update beatmap metadate to indicate beatmap state
```

Comment:
Usually with comments, we try not to document what the code is doing. The code should be clear enough to describe itself. We instead try to focus on *why*  the code does what it does.

With that in mind, I'm curious *why* this change is needed. When would we see this message?

Reply:
Oh, apologies for the error ~

The change was intended to indicate the beatmap load state. If a beatmap is in the process of being load, Metadata.Title can be updated to => "selected beatmap not loaded!", instead of "no beatmaps available!" to provide a more accurate description of the current beatmap state. 

I guess technically speaking, "no beatmaps available!" still makes sense, since the beatmap itself hasn't become available yet.